### PR TITLE
[JRO] Rake task to re-seed database

### PR DESCRIPTION
**Pull request by jayroh**
This commit allows us to quickly and easily truncate database tables, then
re-seed with new sample data. This can, and will, be used in the demo
application to blow away data and re-create it on the daily.

Addresses Issue #245

### CODE REVIEW

**Comment by glebm** [comment]
Why not just run this daily:

```
rake db:drop db:create db:migrate db:seed
```

**Comment by jayroh** [comment]
This would skip drop, create, and migrate completely.

And - I'd probably use this for my dev environment pretty regularly too so the less I have to do, the better 😛

**Comment by glebm** [comment]
It only shaves off 1-3 seconds but sure :+1:

File: Rakefile [hunk 1]
```diff
@@ -133,3 +133,27 @@ else
...
   task default: [:spec, :rubocop]
 end
+
+namespace :db do
+  desc "Wipe out all tables' data"
+  task truncate: :environment do
+    adapter = ActiveRecord::Base.configurations[::Rails.env]['adapter']
+    connection = ActiveRecord::Base.connection
+
+    tables = connection.tables.select do |table|
```
Comment (glebm) [inline Rakefile]:
```ruby
connection.tables - ['schema_migrations']
```

File: Rakefile [hunk 2]
```diff
@@ -133,3 +133,27 @@ else
...
+      table.exclude?('schema_migrations')
+    end
+
+    tables.each do |table|
+      if adapter.include? 'sqlite'
```
Comment (glebm) [inline Rakefile]:
```ruby
connection.adapter_name =~ /sqlite/i
```

**Pull request merged**